runtime/onert/backend/cpu/BackendContext.cc

```diff
@@ -37,7 +37,7 @@ FunctionMap BackendContext::genKernels()
 {
   FunctionMap ret;
 
-  for (auto op_ind : _data.op_order)
+  for (auto &op_ind : _data.op_order)
```

Comment:
(Optional)
```suggestion
  for (auto &&op_ind : _data.op_order)
```

"It is safe, and in fact, preferable in generic code, to use deduction to forwarding reference, for (auto&& var : sequence)." in https://en.cppreference.com/w/cpp/language/range-for.

Reply:
@ragmani I'll update this.